--- a/lib/Net/XMPP/Debug.pm.orig	Fri Apr 18 16:16:37 2014
+++ b/lib/Net/XMPP/Debug.pm	Fri Apr 18 16:17:43 2014
@@ -151,7 +151,7 @@
     my %args;
     while($#_ >= 0) { $args{ lc pop(@_) } = pop(@_); }
 
-    delete($args{file}) if (lc($args{file}) eq "stdout");
+    delete($args{file}) if (defined $args{file} && lc($args{file}) eq "stdout");
 
     $args{time} = 0 if !exists($args{time});
     $args{setdefault} = 0 if !exists($args{setdefault});
